### PR TITLE
hw9 with files was added

### PR DESCRIPTION
**Pull request by serhii-k-n**

### CODE REVIEW

File: Homework_9/except.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+Юзер вводить з консолі ключ, за яким треба отримати значення.
+Написати програму, яка імплементує дану задачу з використанням блоку try/except (Треба обробити помилку якщо ключа не існує).
+"""
+course = {'title': 'AQA', 'language': 'Python', 'level': 'PRO', 'frame': 'pytest'}
+user_key = input('Write a key from a dictionary: ').lower()
+
+try:
+    print(f'The value key \'{user_key}\' is: {course[user_key]}')
+except KeyError:
+    print(f'There is no key \'{user_key}\' in a dictionary')
```
Comment (MarinaZelenska) [inline Homework_9/except.py:13]: Also as variant
image

File: Homework_9/file_txt.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+"""
+
+with open('first.txt', 'w') as first_file:
+    first_file.write(my_str)
+
+with open('first.txt', 'r') as first_file:
+    data = first_file.read()
+
+with open('second.txt', 'w') as second_file:
+    second_file.write(data)
```
Comment (MarinaZelenska) [inline Homework_9/file_txt.py:15]: Can be also
image

**Review by MarinaZelenska — APPROVED** [review]

File: Homework_9/except.py (hunk 1 above)
Comment (serhii-k-n) [reply Homework_9/except.py:13]: Like it, thanks, done

File: Homework_9/file_txt.py (hunk 2 above)
Comment (serhii-k-n) [reply Homework_9/file_txt.py:15]: it does not work on my PyCharm

**Pull request merged**